slides/01_intro.qmd

```diff
@@ -1,6 +1,7 @@
 ---
 title: "1 Introduction"
-author: "Ya, Laura, Matt"
+subtitle: "Creators: Daniel Sabanes Bove, Friedrich Pahlke, Kevin Kunzmann, Liming Li, Joe Zhu, Shuang Li"
```

Comment:
```suggestion
```

Reply:
Since we have the license information slide at the end, we don't need to list creators here.